lib/Ohmbrewer_Menu/Ohmbrewer_Menu_WiFi.cpp

```diff
@@ -12,7 +12,7 @@
     //Set up list of options in vector
     options.push_back("WiFi On");
     options.push_back("WiFi Off");
-    _selectedOption = _settings->isWifiOn();
+    _selectedOption = 0;
```

Comment:
No, I wanted it to start at the first option every time. I think thats how I would expect a menu to work.